files/angular-pouchdb/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-pouchdb",
+  "repo": "angular-pouchdb/bower-angular-pouchdb",
+  "files": {
+    "include": ["angular-pouchdb.js", "angular-pouchdb.min.js"]
```

Comment:
It's here: https://github.com/angular-pouchdb/bower-angular-pouchdb/tree/v3.0.0. Should the `github` property in `info.ini` point to bower-angular-pouchdb?